### PR TITLE
feat(Dashboard): discover button to the new discover page

### PR DESCRIPTION
**Pull request by marcovincit**
### Remove the Discover button in the sidebar
This item is out of context in this place, all items here are relative to things owned by the user.

| Old      | New |
| ----------- | ----------- |
| image      | image       |

### Remove the Discover button in the user menu

| Old      | New |
| ----------- | ----------- |
| image      | image       |

### Add Discover button in the header
Move to the new place promoting the new Discover page: https://codesandbox.io/discover

##### Old
image

##### New
image

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Header/index.tsx [hunk 1]
```diff
@@ -121,6 +121,22 @@ export const Header: React.FC<HeaderProps> = React.memo(
             Create
           </Button>
 
+          <Button
+            variant="ghost"
+            css={css({ width: 'auto' })}
```
Comment (olarclara) [inline packages/app/src/app/pages/Dashboard/Header/index.tsx:126]:
```suggestion
            autoWidth
```

File: packages/app/src/app/pages/Dashboard/Header/index.tsx [hunk 2]
```diff
@@ -121,6 +121,22 @@ export const Header: React.FC<HeaderProps> = React.memo(
...
+            onClick={() => {
+              window.open('http://codesandbox.io/discover', '_blank');
+            }}
+          >
+            <Icon
+              name="discover"
+              size={22}
+              title="New"
+              css={css({ paddingRight: 2 })}
```
Comment (olarclara) [inline packages/app/src/app/pages/Dashboard/Header/index.tsx:135]:
```suggestion
              css={{ paddingRight: 2 }}
```

We've dropped the `css` from new code to make migration to the v2 codebase easier.

File: packages/app/src/app/pages/Dashboard/Header/index.tsx [hunk 3]
```diff
@@ -121,6 +121,22 @@ export const Header: React.FC<HeaderProps> = React.memo(
             Create
           </Button>
 
+          <Button
+            variant="ghost"
+            css={css({ width: 'auto' })}
+            onClick={() => {
+              window.open('http://codesandbox.io/discover', '_blank');
```
Comment (olarclara) [inline packages/app/src/app/pages/Dashboard/Header/index.tsx:128]:
```suggestion
              window.open(`http://codesandbox.io${dashboardUrls.discover(activeTeam)}`, '_blank');
```

**Review by olarclara — COMMENTED** [review]
I think we should add a ticket to revisit the mobile header:

uqn228-3000 preview csb app_dashboard_recent_workspace=8720bfba-1924-427f-8430-d607b305eb6a

File: packages/app/src/app/pages/Dashboard/Header/index.tsx (hunk 3 above)
Comment (marcovincit) [reply packages/app/src/app/pages/Dashboard/Header/index.tsx:128]: @olarclara this will redirect to the website?

**Review by olarclara — APPROVED** [review]

**Review by lenasmayor — APPROVED** [review]

**Review by filipelima18 — APPROVED** [review]

File: packages/app/src/app/pages/Dashboard/Header/index.tsx (hunk 3 above)
Comment (olarclara) [reply packages/app/src/app/pages/Dashboard/Header/index.tsx:128]: dropped the commit that accepted the suggestion, my bad.

**Pull request merged**